articles/service-fabric/how-to-managed-cluster-availability-zones.md

```diff
@@ -218,6 +218,32 @@ Requirements:
    ```
    If you run in to any problems reach out to support for assistance.
 
+## Enable FastZonalUpdate on Service Fabric managed clusters (preview)
+Service Fabric managed clusters support faster cluster and application upgrades by reducing the max upgrade domains per availability zone from five to three.  
+
+The update should be done via ARM template by setting the fastZonalUpdate property to “fast”, adding a node and then removing the node.  The Service Fabric managed cluster resource apiVersion should be 2023-01-01-preview or later.
+
+1. Modify the ARM template with the new property mentioned above.
+```json
+   "resources": [
+        {
+            "type": "Microsoft.ServiceFabric/managedClusters",
+            "apiVersion": "2023-01-01-preview",
+            '''
+            "properties": {
+                '''
+                "zonalResiliency": true,
+                "zonalUpdateMode": “fast”,
+                ...
+            }
+        }]
+```
+2. Add a node to the node type from a cluster by following the procedure to [modify node type](./how-to-managed-cluster-modify-node-type.md) 
```

Comment:
The description for for step 2 & 3 is not accurate. 

We need to convey that customers need to execute a "nodetype" update for every nodetype, so that the UDs on them are reconfigured to reflect the new configuration. 
The update itself could be any change on the nodetype property.

Reply:
thanks.  Added the info.

Reply:
updated.